Hello.html

```diff
@@ -0,0 +1,13 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
Great job @Jaymelfah 🤝

- [ ] Kindly link your HTML file with the `Hello.css` file correctly. The styles are not reflecting well on the text.